dll/win32/shell32/droptargets/CRecyclerDropTarget.cpp

```diff
@@ -109,8 +109,8 @@ class CRecyclerDropTarget :
             cfShellIDList = RegisterClipboardFormatW(CFSTR_SHELLIDLIST);
         }
 
-        HRESULT WINAPI DragEnter(IDataObject *pDataObject,
-                                            DWORD dwKeyState, POINTL pt, DWORD *pdwEffect)
+        STDMETHOD(DragEnter)(IDataObject *pDataObject,
```

Comment:
Refering to the other PR where I mentioned an article,
https://microsoft.public.vc.atl.narkive.com/j1pgALCm/stdmethod-vs-stdmethodimp#post2
> [...] Thus STDMETHOD must be used for declaring new methods (e.g. in
the interface definition produced by MIDL compiler), while STDMETHODIMP
must be used when implementing the method in .cpp file. Either can be
used when declaring a method in your class derived from the interface
('virtual' keyword is optional there). [...]

Here we are in the case of using `STDMETHODIMP`.

Reply:
Fixed.